metadata-proxy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google_containers/nginx
```

Comment:
Ok, I'm following fluentd's lead and using "gcr.io/google-containers/debian-base-amd64:0.1".